.github/workflows/test.yml

```
@@ -0,0 +1,24 @@
+name: Tests
+on: [push, pull_request]
+
+jobs:
+  lint:
```

```suggestion
  test:
```

On mettra les tests unitaires etc après ESLint dans le même job